person.rb

```diff
@@ -24,6 +28,11 @@ def correct_name
     @name
   end
 
+  def class1=(class1)
+    @class1 = class1
+    class1.students.push(self) unless class1.students.include?(self)
+  end
```

Comment:
- [ ] There is no relationship between the Person class and the Classroom class. Please remove this method. There is just a relationship between the Classroom class and the Student class.